src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelTests.kt

```diff
@@ -0,0 +1,174 @@
+package com.terraformation.backend.seedbank.model.accession
+
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.seedbank.AccessionId
+import com.terraformation.backend.db.seedbank.AccessionState
+import com.terraformation.backend.db.seedbank.DataSource
+import com.terraformation.backend.db.seedbank.ProcessingMethod
+import com.terraformation.backend.db.seedbank.SeedQuantityUnits
+import com.terraformation.backend.db.seedbank.ViabilityTestId
+import com.terraformation.backend.db.seedbank.ViabilityTestResultId
+import com.terraformation.backend.db.seedbank.ViabilityTestSubstrate
+import com.terraformation.backend.db.seedbank.ViabilityTestType
+import com.terraformation.backend.db.seedbank.WithdrawalId
+import com.terraformation.backend.db.seedbank.WithdrawalPurpose
+import com.terraformation.backend.seedbank.grams
+import com.terraformation.backend.seedbank.model.AccessionModel
+import com.terraformation.backend.seedbank.model.SeedQuantityModel
+import com.terraformation.backend.seedbank.model.ViabilityTestModel
+import com.terraformation.backend.seedbank.model.ViabilityTestResultModel
+import com.terraformation.backend.seedbank.model.WithdrawalModel
+import com.terraformation.backend.seedbank.seeds
+import java.time.Clock
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneOffset
+
+abstract class AccessionModelTests {
```

Comment:
Might be worth having a discussion about how we want to name this kind of thing, but the existing abstract test superclass in the code base uses a singular name (`DatabaseTest`) rather than a plural one. To stay consistent until we make a decision to use some other naming convention, can we call this `AccessionModelTest` instead? That name is free now!

Reusing that class name shouldn't cause confusion in git history since the original `AccessionModelTest` was in a different directory.

Reply:
I'd thought plural since it's an umbrella for those other test classes, but of course each of those is itself an "umbrella" for multiple tests. Singular sounds good to me!